proto/v1beta1/discovery.proto

```diff
@@ -84,4 +84,7 @@ message Discovered {
   // details to show to the user. The LocalizedMessage is output only and
   // populated by the API.
   google.rpc.Status analysis_status_error = 4;
+
+  // Output only. The time occurrences related to this discovery occurrence were archived.
+  google.protobuf.Timestamp archive_time = 5;
```

Comment:
Let's add this field only in V1 and not in v1beta1.

Reply:
👍 , done